app/routes/app._index.jsx

```diff
@@ -1,44 +1,51 @@
-import { json } from "@remix-run/node";
-import { useLoaderData, Link, useNavigate } from "@remix-run/react";
-import { authenticate } from "../shopify.server";
-import {
-  Card,
-  EmptyState,
-  Layout,
-  Page,
-  IndexTable,
-  Thumbnail,
-  Text,
-  Icon,
-  InlineStack,
-} from "@shopify/polaris";
+import { useLoaderData, useNavigate } from "react-router";
+import { boundary } from "@shopify/shopify-app-react-router/server";
+import { ImageIcon } from "@shopify/polaris-icons";
 
+import { authenticate } from "../shopify.server";
 import { getQRCodes } from "../models/QRCode.server";
-import { AlertDiamondIcon, ImageIcon } from "@shopify/polaris-icons";
 
 // [START loader]
 export async function loader({ request }) {
   const { admin, session } = await authenticate.admin(request);
   const qrCodes = await getQRCodes(session.shop, admin.graphql);
 
-  return json({
+  return {
     qrCodes,
-  });
+  };
 }
 // [END loader]
 
 // [START empty]
-const EmptyQRCodeState = ({ onAction }) => (
-  <EmptyState
-    heading="Create unique QR codes for your product"
-    action={{
-      content: "Create QR code",
-      onAction,
-    }}
-    image="https://cdn.shopify.com/s/files/1/0262/4071/2726/files/emptystate-files.png"
-  >
-    <p>Allow customers to scan codes and buy products using their phones.</p>
-  </EmptyState>
+const EmptyQRCodeState = () => (
+  <s-section accessibilityLabel="Empty state section">
+    <s-grid gap="base" justifyItems="center" paddingBlock="large-400">
+      <s-box maxInlineSize="200px" maxBlockSize="200px">
+        <s-image
+          aspectRatio="1/0.5"
+          src="https://cdn.shopify.com/s/files/1/0262/4071/2726/files/emptystate-files.png"
+          alt="A stylized graphic of a document"
+        />
+      </s-box>
+      <s-grid justifyItems="center" maxBlockSize="450px" maxInlineSize="450px">
+        <s-heading>Create unique QR codes for your products</s-heading>
+        <s-paragraph>
+          Allow customers to scan codes and buy products using their phones.
+        </s-paragraph>
+        <s-stack
+          gap="small-200"
+          justifyContent="center"
+          padding="base"
+          paddingBlockEnd="none"
+          direction="inline"
+        >
+          <s-button href="/app/qrcodes/new" variant="primary">
+            Add QR code
```

Comment:
Any reason why we went with "Add" as opposed to "Create"? I notice we say "Create" above (on line 31) and the Shopcodes app uses the word "Create" as the main action rather than "Add".

<img width="849" alt="26-39-qjavf-hnhu1" src="https://github.com/user-attachments/assets/c5c6fe01-0f68-48c0-baff-2ab1f84c1874" />